compiler/rustc_hir/src/intravisit.rs

```diff
@@ -729,7 +726,12 @@ pub fn walk_expr<'v, V: Visitor<'v>>(visitor: &mut V, expression: &'v Expr<'v>)
         ExprKind::DropTemps(ref subexpression) => {
             visitor.visit_expr(subexpression);
         }
-        ExprKind::Let(ref let_expr) => visitor.visit_let_expr(let_expr),
+        ExprKind::Let(ref let_expr) => {
```

Comment:
Nit: could you match on struct fields too, to ensure we catch changes to the HIR tree?